TileFormats/VectorData/README.md

```diff
@@ -1,20 +1,207 @@
+## Overview
 
-Could also name this a `Geometry` tile.
+The _Vector_ tile format allows streaming of vector datasets including points, polylines, and polygons.  Points can be represented with a combination of billboard, label, and point graphics primitives.
 
-**Implementation work-in-progress**
-* Polygons: https://github.com/AnalyticalGraphicsInc/cesium/pull/4186
-* Polylines: https://github.com/AnalyticalGraphicsInc/cesium/pull/4208
+Each point, poyline, and polygon is a _feature_ in the 3D Tiles specification language. 
+
+## Layout
+
+A tile is composed of a header section immediately followed by a body section.
+
+**Figure 1**: Vector layout (dashes indicate optional fields). 
+
+![header layout](figures/header-layout.png)
+
+## Header
+
+The 28-byte header contains the following fields:
+
+| Field name | Data type | Description |
+| --- | --- | --- |
+| `magic` | 4-byte ANSI string | `"vctr"`. This can be used to identify the arraybuffer as a Vector tile. |
+| `version` | `uint32` | The version of the Vector Data format. It is currently `1`. |
+| `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. If `featureTableJSONByteLength` is zero, this will also be zero. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. | 
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed of two fields: `Feature Table` and `Batch Table`.
+
+Code for reading the header can be found in
+[Vector3DModelTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Vector3DModelTileContent.js)
+in the Cesium implementation of 3D Tiles.
+
+## Feature Table
+
+Contains values for `vctr` semantics used to render features.  The general layout of a Feature Table is described in the [Feature Table specification](../FeatureTable).
+
+The `vctr` Feature Table JSON schema is defined in [vctr.featureTable.schema.json](../../schema/vctr.featureTable.schema.json).
+
+TODO: Write schema after this gets looked over a bit.
+
+### Semantics
+
+If a semantic has a dependency on another semantic, that semantic must be defined as well.
+If both `POSITION` and `POSITION_QUANTIZED` are defined, the higher precision `POSITION` will be used.
+Per-feature semantics specific to a feature type are prefixed with the name of the feature type. e.g. `POLYGON` for polygons and `POLYLINE` for polylines.
+
+At least one global `LENGTH` semantic must be defined. 
```

Comment:
Here and below in the `Global Semantics` table, I don't think we need to require this.  Throughout 3D Tiles if length is zero (e.g., no feature table), the tile doesn't need to be rendered.  Requiring that one of these be defined still does not guarantee that one is non-zero so it is not really a meaningful requirement.